src/main/java/ome/codecs/JPEGCodec.java

```diff
@@ -82,7 +90,47 @@ public byte[] compress(byte[] data, CodecOptions options)
       options.bitsPerSample / 8, false, options.littleEndian, options.signed);
 
     try {
-      ImageIO.write(img, "jpeg", out);
+      //ImageIO.write(img, "jpeg", out);
+        
+      //::phaub 09.02.23   (Adjustable jpeg quality)
+      	
+      // How to use:
+      // Set jpegquality as system property in the calling object (e.g. QuPath using OMEPyramidWriter()):
+      //   String OME_JPEGQUALITY = "ome.codec.jpegquality";
+      //   double jpegquality = 0.95;
+      //   System.setProperty(OME_JPEGQUALITY, String.valueOf( jpegquality ));
+    	    	
+      String OME_JPEGQUALITY = "ome.codec.jpegquality";
+    	  
+      double jpegquality = 0.75;
+      String sJPEGQuality = System.getProperty(OME_JPEGQUALITY, null);
```

Comment:
In general :+1: for making this configurable. I went looking to see if I could find a way that this could be passed via `CodecOptions` or `MetadataOptions` so this wouldn't be pinned to the entire JVM. I didn't see a way and assume it would be an API addition to do that.

Reply:
https://github.com/ome/ome-codecs/blob/master/src/main/java/ome/codecs/CodecOptions.java#L113 might be an option?